pyscf/df/outcore.py

```diff
@@ -163,6 +168,7 @@ def cholesky_eri_b(mol, erifile, auxbasis='weigend+etb', dataname='j3c',
     #    cintopt = gto.moleintor.make_cintopt(atm, bas, env, int3c)
     cintopt = gto.moleintor.make_cintopt(atm, bas, env, int3c)
     bufs1 = numpy.empty((comp*max([x[2] for x in shranges]),naoaux))
+    bufs2 = numpy.empty_like(bufs1)
```

Comment:
It seems that the double buffers are not effective as expected. The second buffer can be used in the `transpose` operation.

Reply:
OK, I did a quick test on a ~15G example and observed that there isn't much difference because the kernel write buffer is already buffering writes.

The double-buffering `does` help if you make HDF5 use the 'direct' IO driver.